documentation/DevelopmentProcess/DevelopmentProcess.tex

```diff
@@ -84,28 +85,39 @@
 \section{Introduction}
 \subsection{Project Overview}
 SmartServe is an autonomous table tennis training system for a wide range of table tennis players to aid in diagnosing and improving their performance. The system does so by shooting balls toward the player and detecting successful returns. It can then adapt to the player's weaknesses and expose them by playing more to them over time. This alleviates problems of finding and working with a coach or training many players in a short amount of time. The system will be deemed a success if table tennis players enjoy and can see value in using our system in place of or in tandem to a coach.\\\\
-The development started at the beginning of the Fall 2017 academic turn and will conclude at the end of the Winter 2018 term. The team was assembled in the capstone course for Software and Mechatronics Engineering disciplines.
+The development started at the beginning of the Fall 2017 academic term and will conclude at the end of the Winter 2018 term. The team was assembled from MECHTRON 4TB6/SFWR ENG 4G06 Capstone course for the Software and Mechatronics Engineering disciplines.
 \subsection{Naming Conventions \& Terminology}
 \begin{itemize}
-\item \textbf{Git}: a distributed versioning control system
+\item \textbf{DevOps}: development operations
+\item \textbf{Git}: a version control system for tracking changes and coordinate work among multiple collaborators
```

Comment:
Should say coordinating, however I do want to keep it as a general definition instead of saying the intent of the software itself. If you agree, I would revert this change but if not just make the grammar correction